packages/renderer/src/lib/preferences/ExtensionIcon.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+import Fa from 'svelte-fa';
+import type { ExtensionInfo } from '../../../../main/src/plugin/api/extension-info';
+import { faPuzzlePiece } from '@fortawesome/free-solid-svg-icons';
+
+export let extension: ExtensionInfo;
+
+let icon: string | undefined = undefined;
+$: {
+  if (extension.icon) {
+    if (typeof extension.icon === 'string') {
+      icon = extension.icon;
+    } else if (extension.icon.dark) {
+      // for now use dark theme
+      icon = extension.icon.dark;
+    }
+  }
+}
+$: fade = extension.state !== 'started' ? ' brightness-50' : '';
+</script>
+
+{#if icon}
+  <img src="{icon}" alt="{extension.name}" class="max-w-10 max-h-10 {fade}" />
```

Comment:
should it contain the `aria-label` ?

Reply:
I've found a couple references like the following that say you don't need aria-label (as well) when image has alt text:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

`If you give your <iframe>s a title, your images an alt attributes, and your input's associated <label>s, aria-label is not necessary.`